site/content/contribute/developer-setup.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Developer setup"
+heading: "Set up your development environment"
+description: "Find out how to set up your development environment for building, running, and testing the Mattermost."
```

Comment:
"...testing the Mattermost [what]"? or "testing Mattermost"? (Please apply this update in both places on the page)

Reply:
The Mattermost you know, like The Facebook. 😆